developer/build_plugin.py

```diff
@@ -103,9 +102,12 @@ def _bake_configuration(sg_connection, source_path, target_path, bundle_cache_ro
         raise ValueError("Unable to get a local copy of %s" % cfg_descriptor)
     baked_descriptor = {
         "type": bootstrap_constants.BAKED_DESCRIPTOR_TYPE,
-        "path": local_path
+        "path": local_path,
+        "system_name": cfg_descriptor.system_name,
```

Comment:
i think it's better to use name here - that's the key that is used by most other descriptors (app store, dev, manual supports it).